interface/wx/desktopenv.h

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        desktopenv.h
+// Purpose:     wxDesktopEnv class interface
+// Author:      Igor Korot
+// Created      2019-10-07
+// Copyright:   (c) 2019 wxWidgets development team
+// Licence:     wxWidgets licence
+///////////////////////////////////////////////////////////////////////////////
+
+/**
+    @class wxDesktopEnv
+ 
+    wxDesktopEnv contains the function for moving the file/folder to Trash
+    or Recycle Bin. The call to this function is synchronous, which means that
+    application will wait for the move to finish.
+ 
+    @since 3.1.3
+*/
+class wxDesktopEnv
+{
+public:
+    /**
+        Default constructor.
+     */
+    wxDesktopEnv();
+    
+    /**
+        Moves the file/folder to the Recycle Bin/Trash.
+        
+        @returns true on success
+    */
+    bool MoveToRecycleBin(wxString &fileName);
```

Comment:
This function should be made static so that you don't need to instantiate an entire wxDesktopEnv class to run it. This is the same way that wxDir::Make and wxDir::Remove work, and it makes more sense that way.

Reply:
Never mind, I goofed we can't do that. I would still like to find a way to be able to just call this without having to have the overhead of instantiating the entire class though.

Reply:
I don't really understand why can't we do that. I think it makes sense to just make it a static function, as you suggested.